modules/understanding-agent-install.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent/installing-with-agent.adoc
+
+:_content-type: CONCEPT
+[id="understanding-agent-install_{context}"]
+= Understanding Agent-based Installer
+As a {product-title} user, you can leverage all the advantages of the Assisted Installer hosted service in disconnected environments.
+
+The Agent installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted Service; both are required to perform the first cluster installation.
+
+The `openshift-install agent create image` subcommand generates an ephemeral ISO based on the inputs that you provide. You can choose to provide inputs through the following manifests:
+
+Preferred:
+
+* install-config.yaml
+* agent-config.yaml
+
+or
+
+Optional: ZTP manifests
+
+* cluster-deployment.yaml
+* agent-cluster-install.yaml
+* pull-secret.yaml
+* infraenv.yaml
+* cluster-image-set.yaml
+* nmstateconfig.yaml
+* registries.conf
+* ca-bundle.crt
+
+[id="agent-based-installer-workflow"]
+== Agent-based Installer workflow
+One of the nodes runs the Assisted Service at the start of the boot process and eventually becomes the bootstrap host. This node is called **node 0**.
+The Assisted Service ensures that all the hosts meet the requirements and triggers a {product-title} cluster deployement. All the nodes have the Red Hat Enterprise Linux CoreOS (RHCOS) image written to the disk. The non bootstrap nodes reboot and initiate a cluster deployment.
```

Comment:
an {product-title}

non-bootstrap

s/ Once / When (IBMSG recommends against using Once in this context: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#once)